Aquality.Selenium.Template/Aquality.Selenium.Template/Forms/Pages/MainPage.cs

```diff
@@ -4,7 +4,7 @@ namespace Aquality.Selenium.Template.Forms.Pages
 {
     public class MainPage : BaseAppForm
     {
-        public MainPage() : base(By.XPath("//section[@class='heroMain heroMain--video-paused']"), "Main page")
+        public MainPage() : base(By.XPath("//section[contains(@class,'heroMain heroMain--video-paused')]"), "Main page")
```

Comment:
//section[contains(@class,'heroMain')] ?

Reply:
The locator was updated.